.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 jobs:
   linux-compat:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
add comment like `#latest` to make it clear WHY we're using this particular version